cli/cmd/resource.go

```diff
@@ -0,0 +1,47 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/prometheus/common/log"
```

Comment:
Hmm ... this is not a direct log library but a wrapper around logrus and the implementation doesn't seem to add much. Might be better using the direct library.